### DISCUSSION TITLE
File Info

### DISCUSSION THREAD

**Discussion by Tnctr-altay-24** [a1ive/grub2-filemanager#345] [Ideas]
Hello @a1ive master; I want to ask, can File Info be shown anywhere in the menu when an iso file or any other file is selected?
+ label {id = "__ Fileinfo __" color = "#ffffff"}

**Comment by Tnctr-altay-24** [comment]
hello @a1ive master; If the F key positions in the fm.txt file in the theme are done as follows, the positions of the F keys and icons will remain the same even if the user sets a different resolution.

e.g

```
+ hbox {
    top = 92%
    left = 38%   
    + vbox {
        + label {text = "[F1]" color = "#ffff00"}
        + image {file = "dock/hwinfo.png"}
    }
    + image {file = "dock/space.png"}
}    
    
+ hbox {
    top = 92%
    left = 43%
    + vbox {
        + label {text = "[F2]" color = "#ffff00"}
        + image {file = "dock/fm.png"}
    }
    + image {file = "dock/space.png"}
}

 + hbox {
    top = 92%
    left = 48%  
    + vbox {
        + label {text = "[F3]" color = "#ffff00"}
        + image {file = "dock/boot.png"}
    }
    + image {file = "dock/space.png"}
}

```

**Reply by a1ive** [reply]
Yes. However, if the user sets a lower resolution (e.g. 640x480) then it may not work so well.

**Reply by Tnctr-altay-24** [reply]
I tested 1920x1080 and 800x600 resolutions are fine but not 640x480 resolutions
